Makefile

```diff
@@ -14,10 +14,20 @@ build.local:
 build.base:
 	DOCKER_BUILDKIT=1 docker build -f dockerscripts/Dockerfile.requirements . -t codecov/baseworker:latest --ssh default
 
+build.base_dev:
+	DOCKER_BUILDKIT=1 docker build --build-arg="REQS=requirements.dev" -f dockerscripts/Dockerfile.requirements . -t codecov/baseworker:latest --ssh default
+
+build.dev:
+	DOCKER_BUILDKIT=1 docker build -f dockerscripts/Dockerfile.local . -t codecov/worker:latest --build-arg RELEASE_VERSION="${release_version}" --ssh default
+
```

Comment:
i don't have an opinion on this myself, but i wonder if we should change these `-t` arguments to create different tags for these different builds? in case anybody accidentally pushes the wrong kind of image to dockerhub under the normal name

it means that to use the local shared image of worker you have to edit `docker-compose.yml` but that isn't a huge deal